airflow-core/src/airflow/configuration.py

```diff
@@ -2138,7 +2137,7 @@ def make_group_other_inaccessible(file_path: str):
 
 
 def ensure_secrets_loaded(
-    default_backends: list[str] = DEFAULT_SECRETS_SEARCH_PATH,
+    default_backends: list[str] | None = None,
```

Comment:
The convention here now is:
- If None, its called from someplace other than worker
- If not None, its called from worker (for `[workers][secrets_backend]` config)


Its not very intuitive, could we go back with the previous behaviour?